### DISCUSSION TITLE
Where can I find the raw .dll file? 😭

### DISCUSSION THREAD

**Discussion by aedion123** [winstxnhdw/lc-hax#289] [Q&A] [answered]
I was wondering if there was a raw .dll file, like without cloning the git. (I'm new to this git stuff) Like where is it >:(

**Comment by joep26020** [comment, marked as answer]
> I was wondering if there was a raw .dll file, like without cloning the git. (I'm new to this git stuff) Like where is it >:(

nope. gotta clone it, recursively. just download git bash and do the command

**Reply by aedion123** [reply]
<img width="372" alt="Screenshot 2024-01-22 161311" src="https://github.com/winstxnhdw/lc-hax/assets/104077178/3ea63077-0f4e-4457-b934-2a80b61039d0">
 Do you know how to fix this, it just says "Command not found"

**Reply by winstxnhdw** [reply]
By installing `git`.

**Reply by aedion123** [reply]
I got it now, so where should the raw .dll file be?

**Reply by o777l** [reply]
You need to build it with the "requirements.bat" where you cloned it to. Then the .dll will be found in the bin folder.

**Reply by winstxnhdw** [reply]
It’ll be a lot easier for you if you read the usage section in the README.
